packages/rocketchat-livechat/server/methods/getAnalyticsOverviewData.js

```diff
@@ -0,0 +1,121 @@
+import moment from 'moment';
+
+Meteor.methods({
+	'livechat:getAnalyticsOverviewData'(options) {
+		if (!Meteor.userId() || !RocketChat.authz.hasPermission(Meteor.userId(), 'view-livechat-manager')) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', {
+				method: 'livechat:getAnalyticsOverviewData'
+			});
+		}
+
+		const from = moment(new Date(options.daterange.from));
+		const to = moment(new Date(options.daterange.to));
+
+		const data = options.analyticsOptions.analyticsOverviewOptions;		// getting data format
+
+		switch (options.analyticsOptions.name) {
```

Comment:
The same I wrote here:
https://github.com/RocketChat/Rocket.Chat/pull/11238/files#r201514190